graph/pkg/service/v0/drives.go

```diff
@@ -31,6 +32,14 @@ import (
 
 // GetDrives implements the Service interface.
 func (g Graph) GetDrives(w http.ResponseWriter, r *http.Request) {
+	sanitized := strings.TrimPrefix(r.URL.Path, "/graph/v1.0/")
```

Comment:
Nitpick: It is great that you have a looooong var name, but for me it would be even nicer if it would not only say that it is santinized - but also, what it is, such as `sanitizedPath` or whatever is more go'ish ;-)

Reply:
😄